source/common/grpc/async_client_impl.h

```diff
@@ -49,8 +49,19 @@ class AsyncClientImpl final : public RawAsyncClient {
   friend class AsyncStreamImpl;
 };
 
+template <typename T> class CarryOn {
```

Comment:
[np] I wonder if we could think about a better name than `CarryOn`. My understanding is that these are optional user-defined callbacks that will exist for the lifetime of the `AsyncStreamImpl` object. Perhaps `OptionalStreamCallbacks` or `OptionalUserDefinedStreamCallbacks`?

Reply:
Bump on the renaming 😄

Reply:
Yup. I renamed in my latest patch. Does it look good to you? I replied to the message in VSCode plugin, but it didn’t work somehow.